inc/class-group.php

```diff
@@ -51,7 +51,16 @@ public function add_page( string $id, Page $page ) {
 	 * @return Page[]
 	 */
 	public function get_pages() : array {
-		return $this->pages;
+		// Sort pages by their order.
+		$pages = $this->pages;
+		uasort( $pages, function ( Page $a, Page $b ) {
+			$order_a = $a->get_meta( 'order' ) ?? 0;
+			$order_b = $b->get_meta( 'order' ) ?? 0;
+
+			return $order_a <=> $order_b;
```

Comment:
This is [PHP 7.0+](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.spaceship-op), not sure if we have a minimum version set?

Reply:
We don't but should add it to all modules in the `require` section of `composer.json` - lets go with `7.1` which I believe all our stacks are on as a minimum.